site/gatsby-site/src/pages/unsubscribe.js

```diff
@@ -36,7 +36,9 @@ const Unsubscribe = (props) => {
     !subscriptionType ||
     !userId ||
     (subscriptionType == 'incident' && !incidentId) ||
-    (subscriptionType !== 'all' && subscriptionType !== 'incident')
+    (subscriptionType !== 'all' &&
```

Comment:
Are these subscription `types` defined somewhere else?

 If not, it would be nice some definition somewhere, like an enum with comments, which should also probably be listed in the readme?

Reply:
An enum with comments sounds good to me. I will also add the subscription types to the Readme file.

Reply:
Done!